lib/cc/cli/analyze.rb

```diff
@@ -70,7 +72,7 @@ def engines
         @engines ||= config.engine_names.map do |engine_name|
           Engine.new(
             engine_name,
-            engine_registry[engine_name],
+            @dev_mode ? make_registry_entry(engine_name) : engine_registry[engine_name],
```

Comment:
:lipstick: no ternaries.

What do you think about extracting a `#registry_entry` to hold the conditional?